app/controllers/series_controller.rb

```diff
@@ -52,6 +52,7 @@ def show
                                                        subscribable_type: 'Series').first ||
                       current_user.subscriptions.build(subscribable: @series)
     end
+    @works = @works.paginate(page: params[:page])
```

Comment:
You might be able to add the `paginate` method to the end of line 46 instead.

Reply:
very true, idk why i put it down there haha